build.wake

```diff
@@ -154,11 +154,40 @@ def tarball Unit =
     # Identify those sources files to include in the tarball
     require Pass allSources = sources "." `.*`
 
-    def srcs =
+    def skipFolders =
+        (".circleci/", "debian/", ".github/", ".vscode/", Nil)
```

Comment:
I actually explicitly chose not to do something like this because regexes are hard to read.

My variable names above make it immediately clear  when its just a singe file, any instance of that filename, or the entire folder whereas listing out all the regexes requires mental overhead to figure out what each one does

Reply:
ok well its still incorrect and you have to fix that. I also find the regex construction below to be a bit confusing personally.

What about a list of globs, you convert each glob to a regex and you check if any of those regexes match?

Reply:
It makes the `catWith` a bit trickier (probably needs to be re-written as a `foldl`), but there's also `map quote`.